Lib/test/test_code.py

```diff
@@ -317,6 +317,8 @@ def func():
         new_code = code = func.__code__.replace(co_linetable=b'')
         self.assertEqual(list(new_code.co_lines()), [])
 
+    @unittest.skipIf(sys.flags.no_debug_ranges,
```

Comment:
Instead of repeating this, let's make it a flag like `test.support.requires_gzip`